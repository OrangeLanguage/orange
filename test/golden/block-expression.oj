def one = {
    def a = 1;
    a
}