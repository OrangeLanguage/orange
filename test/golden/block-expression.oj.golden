const one = (() => {const a = new _Int(1);a;return a;})();one